scene/main/http_request.h

```diff
@@ -68,12 +68,14 @@ class HTTPRequest : public Node {
 	bool validate_ssl;
 	bool use_ssl;
 	HTTPClient::Method method;
-	String request_data;
+	Vector<uint8_t> request_data;
 
 	bool request_sent;
 	Ref<HTTPClient> client;
 	PackedByteArray body;
 	volatile bool use_threads;
+	volatile bool accept_gzip;
```

Comment:
Doesn't need to be volatile. Not even sure why `use_threads` is volatile actually.